Makefile

```diff
@@ -0,0 +1,6 @@
+build-prod:
+	podman build --tag=quay.io/jkadlcik/log-detective:latest ./docker/production/
+
+# don't forget to `podman login quay.io`
+push-prod:
+	podman push quay.io/jkadlcik/log-detective:latest
```

Comment:
right, missed it, sorry

I get your point. I'd prefer use podman directly because I don't need any daemon for that, for docker-c I need podman/docker daemon running. Will update the PR.